src/IndodanaSecurity.php

```diff
@@ -7,7 +7,7 @@ public static function generateBearerToken($apiKey, $apiSecret)
   {
```

Comment:
Class name should be called `IndodanaAPISecurity`

Reply:
May I know why you include `API`?

Reply:
Because it's generating content that directly related with API security haha

Reply:
Alright. I don't see any issue on your suggestion. Will update it